lib/manager/nvm/extract.ts

```diff
@@ -4,7 +4,7 @@ import { PackageDependency, PackageFile } from '../common';
 export function extractPackageFile(content: string): PackageFile {
   const dep: PackageDependency = {
     depName: 'node',
-    currentValue: content.trim(),
+    currentValue: `v${content.trim().replace(/^v/i, '')}`,
```

Comment:
I used replace here to handle capital V as well. We could be extra liberal and just strip all letter characters from the file.